Gemfile

```diff
@@ -36,6 +37,7 @@ group :development, :test do
   gem 'dotenv-rails'
   gem 'stripe-ruby-mock', '~> 2.3.1', require: 'stripe_mock'
   gem 'faker'
+
```

Comment:
I think that we will need Faker in the main group that will be installed on production. Otherwise we will not be able to run seeds on Heroku.